connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerSourceTask.java

```diff
@@ -139,6 +142,11 @@ public void execute() {
             }
 
             while (!isStopping()) {
+                if (isPaused()) {
+                    Utils.sleep(PAUSE_BACKOFF_MS);
```

Comment:
Any reason to do this rather than a `wait`/`notify` combo? Seems like that would remove the magic constant and allow us to respond immediately to a request to resume the task.

This isn't a deal breaker, but it looks like there's probably a better solution.